test_util/launch.py

```diff
@@ -1,20 +1,282 @@
-"""
+"""DC/OS Launch
 
 Usage:
-dcos-launch create [--wait] [--dump-info=cluster_info.json] <config=config.yaml>
-dcos-launch wait <path=cluster_info.json>
-dcos-launch describe <path=cluster_info.json>
-dcos-launch delete <path=cluster_info.json>
+  dcos-launch create [--config-path=<path>] [--info-path=<path>]
+  dcos-launch wait [--info-path=<path>]
+  dcos-launch describe [--info-path=<path>]
+  dcos-launch test [--info-path=<path>] [--] [<test_command>]...
```

Comment:
This test isn't intended for people not developing DC/OS, while dcos-launch in general _is_ intended for people doing that. I feel somewhat that `test` shouldn't be a command at all on dcos-launch (it's ambiguous to people not actively working on DC/OS internals). I feel very strongly that this not be called `test` at the very least. Something like `run-internal-integration-tests` would make a lot more sense and make it more evident this isn't running arbitrary tests, and isn't shakedown either.